internal/cmd/pipeline/command_create.go

```diff
@@ -40,8 +40,12 @@ func (c *command) newCreateCommand(enableSourceCode bool) *cobra.Command {
 	pcmd.AddClusterFlag(cmd, c.AuthenticatedCLICommand)
 	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
 
-	_ = cmd.MarkFlagRequired("ksql-cluster")
-	_ = cmd.MarkFlagRequired("name")
+	if enableSourceCode {
+		cobra.CheckErr(cmd.MarkFlagFilename("sql-file", "sql"))
```

Comment:
@xiangxin72 Is it safe to assume that `--sql-file` is usually suffixed with .sql?

Reply:
yes it's definitely safe to assume that.